runtime/gc_base/RootScanner.hpp

```diff
@@ -93,7 +93,10 @@ class MM_RootScanner : public MM_BaseVirtual
 	bool _includeJVMTIObjectTagTables; /**< Should the iterator include the JVMTIObjectTagTables. Default true, should set to false when doing JVMTI object walks */
 #if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
 	bool _includeDoubleMap; /**< Enables doublemap should the GC policy be balanced. Default is false. */
-#endif /* J9VM_GC_ENABLE_DOUBLE_MAP */
+#endif /* defined(J9VM_GC_ENABLE_DOUBLE_MAP) */
+#if defined(J9VM_ENV_DATA64)
```

Comment:
like it's the case for J9VM_GC_ENABLE_DOUBLE_MAP, it would be cleaner if we have an explicit compile time flag for this new feature (that would happen to equal to J9VM_ENV_DATA64)

Reply:
Okay. This would mean all my current uses of `J9VM_ENV_DATA64` and `OMR_ENV_DATA64` would need to be modified to this new compile-time flag. This is not a big change, but please confirm that I should go ahead with this change.

Reply:
if there are a lot more beyond the code you already touch here, it could be a separate PR that goes ahead of this